inc/fw-update.php

```diff
@@ -56,7 +56,7 @@ function spbc_security_firewall_update__init( $delay = 0 ){
     $spbc->fw_stats['calls']          = 0;
     $spbc->fw_stats['updating_id']    = md5( mt_rand( 0, 100000 ) );
     $spbc->fw_stats['updating_last_start'] = time();
-    $spbc->save( 'fw_stats' );
+    $spbc->save( 'fw_stats', true, false );
```

Comment:
may be param autoload in save function set false by default?

Reply:
I don't think so, because there will be more to change.
But I can agree that application level doesn't need to know anything about the autoload flag.

Maybe moving this logic to State::save() function will be a good practice. What do you think?

Reply:
I think that you need to understand exactly whether the autoloading option is needed when the plugin is running and already start from this.